src/components/Form/Form.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function Form() {
+  return (
+    <form action="#" className="form">
+      <div className="form-group">
+        <label htmlFor="hello" className="form-label">
+          <input type="text" name="hello" id="hello" className="form-input" />
+        </label>
+      </div>
+    </form>
```

Comment:
- [ ] It would be better if this component Renders a form with inputs for the title and the author of the book and has a button to submit :blush:

Reply:
Done! Thanks